plugins/quay/src/components/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { VulnerabilitySeverity } from '../types';
+
+const byteUnits = ['B', 'kB', 'MB', 'GB', 'TB'];
+
+export const SEVERITY_COLORS = new Proxy(
+  new Map([
+    [VulnerabilitySeverity.Critical, '#7D1007'],
+    [VulnerabilitySeverity.High, '#C9190B'],
+    [VulnerabilitySeverity.Medium, '#EC7A08'],
+    [VulnerabilitySeverity.Low, '#F0AB00'],
+    [VulnerabilitySeverity.None, '#3E8635'],
+  ]) as any,
+  {
+    get: (o: Map<VulnerabilitySeverity, string>, k: VulnerabilitySeverity) =>
+      o.has(k) ? o.get(k) : '#8A8D90',
+  },
+);
+
+export function formatDate(date: string | number): string {
+  if (date === -1) {
+    return 'N/A';
+  }
+
+  const adjustedDate = typeof date === 'number' ? date * 1000 : date;
```

Comment:
Is date here here expressed in seconds and being converted to MS? 
If so, then you can express that with a TS docstring like:
```ts
/**
 * Returns the given date as a formated Date.
 * 
 * @param date - The given date in seconds
 * @return The date formatted to en-US locale
 */
```